.github/workflows/test_coverage.yml

```diff
@@ -51,7 +51,7 @@ jobs:
       - name: install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install -e .[test_full]
+          pip install -e .[test_full] --use-pep517
```

Comment:
this is old code

Reply:
Can't see where it is removed. I guess I need to replace Makefile with modern makefile completely.